packages/aws-cdk/test/api/hotswap/hotswap-test-setup.ts

```diff
@@ -83,8 +83,24 @@ export class HotswapMockSdkProvider {
     });
   }
 
-  public stubLambda(stubs: SyncHandlerSubsetOf<AWS.Lambda>) {
-    this.mockSdkProvider.stubLambda(stubs);
+  public stubLambda(stubs: SyncHandlerSubsetOf<AWS.Lambda>, additionalProperties: { [key: string]: any } = {}) {
+    this.mockSdkProvider.stubLambda(stubs, {
+      api: {
+        waiters: {},
+      },
+      makeRequest() {
+        return {
+          promise: () => Promise.resolve({}),
+          response: {},
+          addListeners: () => {},
+        };
+      },
+      ...additionalProperties,
+    });
+  }
+
+  public getLambdaApiWaiters() {
```

Comment:
Please specify the return type of this method (you might want to introduce some anonymous types here, to help consumers of this method).

Reply:
Added return type of `{ [key: string]: any }`